.github/workflows/macos.yml

```diff
@@ -60,7 +69,14 @@ jobs:
       - name: check
         run: |
           cd _build
-          make check TESTSUITEFLAGS="--jobs=$((${NPROC}+1))"
+          make --jobs=$((${NPROC}+1)) check TESTSUITEFLAGS="--jobs=$((${NPROC}+1))"
```

Comment:
Only a minimal thing: if you did the build before you can do

```sh
cd _build/tests
make check TESTSUITEFLAGS="--jobs=$((${NPROC}+1))"
```

Reply:
Ok; I guess you know best whether the other directories have some `check` targets.